highfidelity/hifi
# Vive: Expose all 16 tracked poses to the controller system

All tracked poses are now accessible to the Controller system via Controller.Hardware.Vive.TrackedObject00 to TrackedObject15.  A test script is included that visualizes the position/orientation of each tracked poses.

QA Test Plan:
  * On the vive connect a third hand controller via USB.
  * run the developer/tests/viveTrackedObjects.js script
  * Enter HMD mode with the vive.
  * You should be able to see the position and orientations of all three hand controllers within interface.
